### PR TITLE
Portfolio: setup and mobile first

### PR DESCRIPTION
**Pull request by Hassaanjbaig-code**
[] This the Portfolio setup project  

   []  Little Description of this project

       [] Add Linters to the project for getting what are problem side by side 
       [] Create Index.html  
           1. Link for the font which is Poppins  form use in the Project 
           2. Add all the necessary tag for the project.
       []  Create style.css
          1.  Added the style according to the requirement.
          2. Mainly use flex in this project 
       [] All background image taken from the give template (Figma)
       [] test it with linters all the error are gone and than upload it

### CODE REVIEW

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Hassaanjbaig-code  😄 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Your linters check did not passed.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Hassaanjbaig-code** [comment]
Hello cand bit explain why It is not working 
What is the problem

**Review by ghost — CHANGES_REQUESTED** [review]
Hi @Hassaanjbaig-code , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Good PR Description👍 


## Required Changes ♻️
image

- [x] I noticed that you had pushed some changes directly into the main branch. Kindly avoid pushing your changes directly into the `main` branch and push them into the current `feature` branch that you are working on. Working with the `feature` branch enables us to review your work properly and it is also a good GitHub flow practice.
- [x] Kindly wrap your toolbar in a `<header>` tag instead of a `<ul>` tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.
- [x] Kindly wrap your logo in an `<a>` tag. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.
- [x] Kindly wrap your hamburger button in a `<nav>` tag for semantic reasons

image

- [ ] Kindly wrap your headline section in a `<section>` tag instead of a `<div>` tag. This represents a generic standalone section of the page

image

- [x] Note that your `social links` section is supposed to be composed of a list of social icons grouped in `<ul>` and `<li>` elements and wrapped in `<a>` tags so that when one clicks there, they are directed to your actual social pages.

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -22,12 +22,16 @@
...
         <li class="left">My logo</li>
+      </a>
+      <nav>
         <li class="right">
           <span class="material-symbols-outlined"> menu </span>
         </li>
-      </ul>
+      </nav>
+      </header>
       <div class="main-text">
```
Comment (HAMMAS-SALEEM) [inline index.html]: - [ ] Kindly use `<section class="main-text">` tag instead of `<div class="main-text">` tag as suggested by the previous reviewer 😊

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @Hassaanjbaig-code :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Github flow
:heavy_check_mark: Following some requirements

Good job on implementing some of the requested changes, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Hassaanjbaig-code) [reply index.html]: Where I use section tags in all or which specific.

**Review by ghost — APPROVED** [review]
Hi @Hassaanjbaig-code , 

Good work implementing the changes. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Good PR Description👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**